patterns/page-01-business-home.php

```diff
@@ -5,7 +5,7 @@
  * Categories: about
  * Keywords: page, starter
  * Block Types: core/post-content
- * Post Types: page
+ * Post Types: page, wp_template
```

Comment:
let's add this to page-portfolio-home too

Reply:
Should probably be on all the page patterns.